src/domain-registry-essentials/architecture-overview.md

```diff
@@ -1,123 +1,135 @@
+
+`Resolver` allows domain owners, approved addresses, and operators to edit records of their owned domains. `Resolver` also provides an interface for getting standardized records, making it easier to access general information about domains.
 
 ### ProxyReader
 
-ProxyReader is a smart contract that is used by our resolution libraries to resolve domains. Normally, it would take at least two queries to Ethereum blockchain to resolve a domain: users would call Registry, to learn a domain's Resolver address, and then they would make a call to Resolver, to read records themselves. ProxyReader provides users with a way to make just one call to get the information they need.
+`ProxyReader` is a smart contract that our [resolution libraries](https://github.com/unstoppabledomains?q=resolution) use to resolve domains. Normally, it would take at least two queries to the Ethereum blockchain to resolve a domain:
+
+1. Call `Registry` to learn a domain's Resolver address
+2. Call `Resolver` to read records themselves
+
+`ProxyReader` consolidates these operations into a single call.
 
-If you're interested in how the resolution process works in more detail, check out [Resolving domain records](resolving-domain-records.md).
+For more information on how the resolution process works read [Resolving Domain Records](resolving-domain-records.md).
 
-## Domain hierarchy and ownership
+## Domain Hierarchy and Ownership
 
-There are plenty of ways of setting up an ownership model in CNS. A domain can be owned by either an external address, that is accessed with a private key or an internal one, i.e. smart contract. Managing domains with smart contracts opens up a lot of possibilities in how to organize ownership. Domain management can be set up to a multi-sig wallet or it can be equally shared among a group of administrators – these are just two simple examples, anything else expressible with Ethereum smart contracts is possible as well.
+Domain ownership can take many forms in CNS.
 
-Subdomains of any level are subjects to the same set of rules as second-level domains in CNS. Subdomains can be transferred or burned by both a direct owner and a domain zone owner \(i.e. parent domain owner\). Only a direct domain owner can set Resolver address and manage domain records.
+A domain can be owned by both an external address (one that is accessed with a private key) or an internal address (i.e.: a smart contract). Managing domains with smart contracts opens up many new ways to structure ownership. For example, domain management could be governed by a multi-signature wallet or it could be equally shared among a group of administrators. These are two simple examples but there are many more possibilities. If it can be expressed in Ethereum smart contracts, it can own a domain.
```

Comment:
> If it can be expressed in Ethereum smart contracts, it can own a domain

Nitpicking: to me, it sounds like Ethereum smart contracts can express rules, but not entities. I think, "it can own a domain" jumps out a bit.

Reply:
Removed the last sentence. Reading through it again I don't think it's necessary to get the point across.